### PR TITLE
feat(resources): new side bar filtering style

### PR DESCRIPTION
**Pull request by runrog**

### CODE REVIEW

File: fonts/config.json [hunk 1]
```diff
@@ -3527,6 +3527,36 @@
...
+      "css": "cc-share",
+      "code": 59416,
+      "src": "entypo"
+    },
+    {
+      "uid": "7132e1233bc16cd1b6efe7e29d3613a5",
+      "css": "linkedin-1",
+      "code": 62232,
+      "src": "entypo"
+    },
```
Comment (WidgetsBurritos) [inline fonts/config.json]: This is different than the original PR. https://github.com/rackerlabs/zoolander/pull/523/files

What is this entypo thing?

Comment (runrog) [reply fonts/config.json]: I added 2 new icons with fontello in the original. The up and down arrow. Not sure how/why a linkdin one came in

Comment (WidgetsBurritos) [reply fonts/config.json]: okay.   it doesn't matter.  I was just curious.

**Comment by WidgetsBurritos** [comment]
Note: there original PR has the other necessary +1.

**Comment by WidgetsBurritos** [comment]
@amberRrucker @runrog @writes,

Since this is the base of #526 I am closing this PR and any changes that have to happen after this need to happen in that PR as a separate commit. This is to help avoid confusion.

**Pull request closed without merging**